google/cloud/bigtable/prepared_query.h

```diff
@@ -48,6 +48,7 @@ class PreparedQuery {
   // constructor as public, but this will be converted to private as
   // originally intended once other classes that orchestrate PreparedQuery
   // are implemented.
+  GOOGLE_CLOUD_CPP_DEPRECATED("use the other constructor")
```

Comment:
Would it be possible to also provide the reasoning behind deprecating this constructor? Perhaps in the PR description.